stockflux-securities-master/src/services/SecuritiesService.js

```diff
@@ -20,3 +20,28 @@ export async function getSecurity(securityId) {
   const security = await response.json();
   return security;
 }
+
+export async function postSecurity(security) {
+  const options = await getWindowOptions();
+  const response = await fetch(
+    `${options.customData.apiBaseUrl}/securities-v2`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(security)
+    }
+  );
+  if (response.status === 201) {
+    const responseJSON = await response.json();
```

Comment:
You could just `return await response.json();` here. No need to assign a variable and then return it.

Reply:
k, changed